code/__HELPERS/game.dm

```diff
@@ -303,8 +303,8 @@ proc/isInSight(var/atom/A, var/atom/B)
 		if(deathtime < 3000 && ( !O.client.holder || !(O.client.holder.rights & R_ADMIN)) )
 			continue
 
-		// Admins and AFK players cannot be drafted
-		if (O.client.inactivity / 600 > ALIEN_SELECT_AFK_BUFFER + 5 || istype(O.client.holder, /datum/admins))
```

Comment:
That may result in some slight complications if an admin unexpectedly ends up being drafted while "bussing" or inspecting. if you could do something about it (if you haven't already), such an alert or similar, it would be good.

Reply:
Added an alert.